docker-compose.yml

```diff
@@ -141,6 +141,7 @@ services:
 volumes:
   node_modules:
   postgres_data:
+  postgres_data_16:
```

Comment:
I like the idea of updating the volume's name! Less confusing for folks pulling down the changes.

What's the thought re: keeping both `postgres_data` and `postgres_data_16` defined here?

Reply:
I had a vague notion about devs possibly wanting to keep and use the old volume, as in 
> some remark about keeping or deleting old volumes
but I think I'll remove it from the docker compose file.